jumpscale/sals/reservation_chatflow/deployer.py

```diff
@@ -503,6 +504,10 @@ def check_farm_capacity(self, farm_name, currencies=None, sru=None, cru=None, mr
             available_mru += node.total_resources.mru - node.reserved_resources.mru
             available_hru += node.total_resources.hru - node.reserved_resources.hru
 
+        farm_id = self._explorer.farms.get(farm_name=farm_name).id
+        gateways = self._explorer.gateway.list(farm_id)
+        running_nodes += len(gateways)
+
         if not running_nodes:
```

Comment:
according to this line 501, I think u should add another variable for gateway check, no?
```
running_nodes += 1
```

Reply:
no. the point is to use the same variable